pkg/build/controller/controller.go

```diff
@@ -208,7 +208,7 @@ func (bc *BuildController) resolveOutputDockerImageReference(build *buildapi.Bui
 		stream, err := bc.ImageStreamClient.GetImageStream(namespace, streamName)
 		if err != nil {
 			if errors.IsNotFound(err) {
-				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist", namespace, streamName)
+				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist.", namespace, streamName)
```

Comment:
> why not just capitalize it here? seems weird to have punctuation but not capitalize the sentence

Errors don't get leading capitals.  I'm not sure why the rule exists, but it does.